source/User_Guide/Marketing_Campaigns/senders.md

```diff
@@ -90,10 +66,7 @@ You can insert your sender identity information into your campaigns using the fo
 
 **[Sender_Zip]** - The sender's zip.
 
-{% anchor h3 %}
-Display Sender Info
-{% endanchor %}
-If you'd like to show your full sender's address and information you can use the tags like so:
+So, to show your full sender's address and information in the footer of the email, you can use the tags like so:
```

Comment:
To show your sender's full address and information in the footer of the email, add the tags as shown: